static/docs/commands-reference/add.md

```diff
@@ -16,31 +16,34 @@ positional arguments:
 ## Description
 
 The `dvc add` command is analogous to the `git add` command. By default an added
-file is committed to the DVC cache. Using the `--no-commit` option, the file
-will not be added to the cache and instead the `dvc commit` command is used when
-(or if) the file is to be committed to the DVC cache.
+file is committed to the <abbr>cache</abbr>. Using the `--no-commit` option, the
+file will not be added to the cache and instead the `dvc commit` command is used
+when (or if) the file is to be committed to the cache.
 
 Under the hood, a few actions are taken for each file in `targets`:
 
 1. Calculate the file checksum.
-2. Move the file content to the DVC cache (default location is `.dvc/cache`).
+2. Move the file content to the cache directory (by default in `.dvc/cache`).
 3. Replace the file by a link to the file in the cache (see details below).
 4. Create a corresponding [DVC-file](/doc/user-guide/dvc-file-format) and store
-   the MD5 checksum to identify the cache entry.
-5. Add the targets to `.gitignore` (if Git is used in this
+   the MD5 checksum to identify the cached file.
+5. Add the `targets` to `.gitignore` (if Git is used in this
    <abbr>workspace</abbr>) to prevent it from being committed to the Git
    repository.
 6. Instructions are printed showing `git` commands for adding the files to a Git
    repository. If a different SCM system is being used, use the equivalent
-   command for that system or nothing is printed if `--no-scm` was specified for
-   the repository.
+   command for that system. Nothing is printed if `--no-scm` was specified when
+   [initializing](/doc/commands-reference/init) the project.
 
-Unless the `-f` options is used, by default the DVC-file name generated is
-`<file>.dvc`, where `<file>` is file name of the first output (from `targets`).
+Note that `targets` outside the current workspace are supported, creating
+[external outputs](/doc/user-guide/external-outputs).
 
-The result is data file is added to the DVC cache, and DVC-files can be tracked
-via Git or other version control system. The DVC-file lists the added file as an
-output (`out`), and references the DVC cache entry using the checksum. See
+Unless the `-f` options is used, the DVC-file name generated is `<file>.dvc` by
+default, where `<file>` is file name of the first output (from `targets`).
+
+The result is data file is placed in the cache directory, and DVC-files can be
+tracked via SCM. The DVC-file lists the added file as an output (`outs` field),
```

Comment:
SCM -> Git?

Reply:
Already fixed, but note that this has its own issue: #540.